### ISSUE TITLE
Alternative breath bubble designs

### ISSUE THREAD

**Issue by BrandonKMLee** [mmazzarolo/breathly-app#87]
As observed in these links, there are alternate breath bubble designs:
https://www.doyou.com/10-awesome-gifs-for-calm-breathing-59450/
https://imgur.com/gallery/Czbn6de

For the breathing nested polygon: https://codepen.io/luizotcarvalho/pen/ogQmgY and https://codepen.io/jhnsnc/pen/EwEqvW
For the breathing star: https://codepen.io/thebabydino/pen/Fxrbd and https://codepen.io/KennyLindahl/pen/waJEOP
For the breathing bubble: https://codepen.io/gnclmorais/pen/wgmvmO and https://oneminch.dev/breathe/
For breathing box: https://codepen.io/thebabydino/pen/gzoBoM and https://reikrom.github.io/work-breath/
For something covering the whole screen: https://github.com/trentmwillis/deepbreathing

**Comment by mmazzarolo** [comment]
Yep, allowing to select different breathing patterns should be straightforward on v2. I think we can start by adding back the old "circle" design and start from there.

**Comment by BrandonKMLee** [comment]
Thanks for the forethought on the design! There are a lot of ways to accessorize the look
